### PR TITLE
adding wrapper sample

### PR DESCRIPTION
**Pull request by dgkanatsios**
Adding the wrapper sample so that users can easily explore MPS by
- using their game server as is (i.e. not integrating with GSDK)
- using a fakegameserver provided in the sample

### CODE REVIEW

File: wrappingGsdk/wrapper/Program.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+{
+    class Program
+    {
+        private static Process gameProcess;
+        static void Main(string[] args)
+        {
+            if(args.Length <= 1 || args.Length >= 3 || args[0] != "-g")
+            {
+                Console.WriteLine("Usage: wrapper.exe -g fakegame.exe");
+                return;
```
Comment (andhesky) [inline wrappingGsdk/wrapper/Program.cs:18]: probably want a way to pass arguments to the wrapped game

File: wrappingGsdk/wrapper/Program.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                IDictionary<string, string> activeConfig = GameserverSDK.getConfigSettings();
+
+                if (activeConfig.TryGetValue(GameserverSDK.SessionCookieKey, out string sessionCookie))
+                {
+                    LogMessage($"The session cookie from the allocation call is: {sessionCookie}");
+                }
+            }
+            else
+            {
+                // No allocation happened, the server is getting terminated (likely because there are too many already in standing by)
```
Comment (andhesky) [inline wrappingGsdk/wrapper/Program.cs:65]: >// [](start = 16, length = 2)

probably want to hard kill the process here.

File: wrappingGsdk/fakegame/Program.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace fakegame
+{
+    class Program
+    {
+        async static Task Main(string[] args)
+        {
```
Comment (andhesky) [inline wrappingGsdk/fakegame/Program.cs:9]: maybe we should have a more interesting game one that at least responds to a request like WinRunner

File: wrappingGsdk/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+> This sample and corresponding technique is NOT meant to be used in production but only for evaluation/development purposes. Proper integration of your game server with the GSDK is highly recommended.
+
+## Wrapping an existing game server using the wrapper app
+
+To get started, you can find two .NET Core projects in the current folder. 
+- `wrapper` is a .NET Core console application that acts as a wrapper for your game server and integrates with GSDK using the [latest Nuget package](https://www.nuget.org/packages/com.playfab.csharpgsdk)
+- `fakegame` is a .NET Core console application that keeps itself alive for ever. It's meant to simulate a game server that has absolutely zero knowledge of GSDK. You can use it if you don't have a game server of your own.
+
+### Building the wrapper
```
Comment (andhesky) [inline wrappingGsdk/README.md:16]: consider a powershell script that does all this (builds both apps and creates a zip file).

**Comment by dgkanatsios** [comment]
@andhesky thanks for the review, PTAL at the updated commit.

File: wrappingGsdk/fakegame/Program.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+
+namespace fakegame
+{
+    class Program
+    {
+        const int HTTP_PORT = 80;
+        static void Main(string[] args)
```
Comment (andhesky) [inline wrappingGsdk/fakegame/Program.cs:16]: >args [](start = 34, length = 4)

Maybe take port as cmd line arg to show it being passed all the way through

File: wrappingGsdk/README.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+```json
+...
+"AssetDetails": [
+    {
+        "MountPath": "C:\\Assets",
+        "SasTokens": null,
+        "LocalFilePath": "C:\\projects\\gsdkSamples\\wrappingGsdk\\drop\\gameassets.zip"
+    }
+],
+"StartGameCommand": "C:\\Assets\\wrapper.exe -g C:\\Assets\\fakegame.exe",
```
Comment (andhesky) [inline wrappingGsdk/README.md:53]: >eg [](start = 63, length = 2)

Maybe show how the port can be passed through on the cmd line:

"StartGameCommand": "C:\\Assets\\wrapper.exe -g C:\\Assets\\fakegame.exe 80"

**Review by andhesky — APPROVED** [review]
:shipit:

**Pull request merged**